src/main/java/org/embl/mobie/io/ome/zarr/util/UnitTypes.java

```diff
@@ -0,0 +1,73 @@
+package org.embl.mobie.io.ome.zarr.util;
+
+public enum UnitTypes {
```

Comment:
@constantinpape @KateMoreva @joshmoore
Would it be better to depend on some other library for this?

Reply:
Yes, although I think there is nothing wrong with a double-check right now. But it is better to move it to `org.janelia.saalfeldlab.n5` once v4 becomes stable.